src/sentry/toolbar/views/__init__.py

```diff
@@ -0,0 +1,14 @@
+TOOLBAR_CSP_SCRIPT_SRC = ["'unsafe-inline'", "sentry.io", "*.sentry.io"]
```

Comment:
This won't work for self-hosted customers.  How do we support their domain?  I'm wondering if there's a setting in `server.conf` or elsewhere that has this value.